cmd/nerdctl/run_network.go

```diff
@@ -164,12 +168,72 @@ func generateNetOpts(cmd *cobra.Command, dataStore, stateDir, ns, id string) ([]
 				ports = append(ports, pm...)
 			}
 		}
+	case nettype.Container:
+		if runtime.GOOS != "linux" {
+			return nil, nil, "", nil, fmt.Errorf("currently '--network=container:<container>' can only works on linux")
+		}
+		if rootlessutil.IsRootless() {
+			return nil, nil, "", nil, fmt.Errorf("currently '--network=container:<container>' can't run rootless")
+		}
+		if len(netSlice) > 1 {
+			return nil, nil, "", nil, fmt.Errorf("only one network allowed using '--network=container:<container>'")
+		}
```

Comment:
the networked container does not work with port mapping, please check it.

Reply:
👌

Reply:
the 5000:5000 need to be published(-o) with the `2f0` container.

checked with docker and prompted `conflicting options: port publishing and the container type network mode.`